packages/core/src/hydration/utils.ts

```diff
@@ -241,10 +247,35 @@ export function processTextNodeMarkersBeforeHydration(node: HTMLElement) {
  * Internal type that represents a claimed node.
  * Only used in dev mode.
  */
-type ClaimedNode = {
-  __claimed?: boolean;
+export enum HydrationStatus {
+  Hydrated = 'hydrated',
+  Skipped = 'skipped',
+  Mismatched = 'mismatched',
+}
+
+export type HydrationInfo =|{
```

Comment:
Can we remove `|` here or it's added by auto-formatting?
```suggestion
export type HydrationInfo = {
```

Reply:
Yes, it's the formatter at it again !

Reply:
Hm, can we disable formatting for this line?